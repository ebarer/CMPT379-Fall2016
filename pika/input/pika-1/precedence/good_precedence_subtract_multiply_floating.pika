exec {

  print 3.8 - 1.7 * 9.6, _n_.
  print 8.0 - 7.1 * 6.0, _n_.
  print 7.7 - 7.2 * 2.2, _n_.
  print 6.6 - 1.4 * 3.5, _n_.
  print 9.9 - 1.1 * 3.4, _n_.
  print 6.2 - 3.0 * 5.2, _n_.
  print 8.1 - 5.1 * 4.8, _n_.
  print 8.0 - 2.9 * 6.9, _n_.
  print 7.0 - 7.4 * 3.5, _n_.
  print 9.9 - 6.6 * 9.6, _n_.
  print 8.4 - 5.7 * 2.8, _n_.
  print 1.0 - 8.4 * 8.0, _n_.

  print 6.6 * 1.5 - 7.3, _n_.
  print 6.8 * 2.7 - 6.7, _n_.
  print 2.0 * 3.5 - 7.5, _n_.
  print 5.7 * 2.9 - 5.1, _n_.
  print 2.8 * 4.6 - 1.4, _n_.
  print 9.8 * 1.6 - 7.9, _n_.
  print 9.1 * 5.1 - 4.4, _n_.
  print 6.8 * 4.3 - 6.1, _n_.
  print 5.8 * 9.1 - 9.4, _n_.
  print 5.9 * 1.8 - 2.9, _n_.
  print 6.6 * 4.3 - 5.2, _n_.
  print 2.8 * 9.2 - 9.1, _n_.

  print 9.9 - 4.9 * 5.9 - 1.0, _n_.
  print 6.3 - 4.6 * 4.1 - 1.9, _n_.
  print 1.1 - 9.6 * 9.7 - 1.4, _n_.
  print 9.9 - 2.7 * 4.5 - 7.1, _n_.
  print 8.8 - 9.1 * 8.2 - 4.8, _n_.
  print 6.5 - 6.1 * 6.0 - 8.5, _n_.
  print 8.8 - 6.0 * 6.0 - 9.0, _n_.
  print 1.7 - 3.7 * 4.6 - 6.3, _n_.
  print 1.2 - 9.9 * 1.7 - 5.6, _n_.
  print 9.3 - 8.5 * 5.2 - 8.5, _n_.
  print 1.8 - 4.2 * 5.4 - 6.1, _n_.
  print 7.5 - 5.8 * 3.3 - 9.9, _n_.

  print 7.6 * 9.6 - 8.6 * 8.2, _n_.
  print 2.3 * 9.7 - 9.2 * 7.4, _n_.
  print 2.3 * 1.6 - 2.0 * 4.2, _n_.
  print 5.4 * 6.5 - 3.8 * 1.7, _n_.
  print 7.4 * 3.6 - 2.7 * 1.1, _n_.
  print 3.3 * 5.9 - 4.6 * 9.1, _n_.
  print 7.1 * 5.8 - 2.3 * 6.5, _n_.
  print 4.6 * 2.6 - 3.0 * 7.7, _n_.
  print 2.7 * 5.4 - 6.7 * 5.7, _n_.
  print 8.1 * 9.3 - 9.2 * 1.9, _n_.
  print 8.1 * 2.9 - 1.3 * 1.4, _n_.
  print 1.5 * 2.2 - 2.6 * 6.1, _n_.

}

