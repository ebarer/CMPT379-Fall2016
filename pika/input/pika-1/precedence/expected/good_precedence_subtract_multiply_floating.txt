-12.52
-34.6
-8.14
1.7
6.16
-9.4
-16.38
-12.01
-18.9
-53.46
-7.56
-66.2
2.6
11.66
-0.5
11.43
11.48
7.78
42.01
23.14
43.38
7.72
23.18
16.66
-20.01
-14.46
-93.42
-9.35
-70.62
-38.6
-36.2
-21.62
-21.23
-43.4
-26.98
-21.54
2.44
-45.77
-4.72
28.64
23.67
-22.39
26.23
-11.14
-23.61
57.85
21.67
-12.56
